### PR TITLE
Change example email domains and account ids

### PR DESCRIPTION
**Pull request by sbkok**
**Why?**

To ensure that email addresses or account ids are not exposed by accident,
this is the first step to ensure that the same example email addresses
and account ids are used. Such that a mechanism can be introduced to
prevent accidentally committing email addresses or account ids that are
not supposed to be share publicly.

**What?**

This step updates all the email addresses and account ids in the code and
documentation.

The instructions for our contributors received an update too.

In case you want to specify an example email address, please make use of
one of the following email addresses:

* `jane@example.com`
* `john@example.com`
* `janes_team@example.com`
* Or another `@example.com`
* For tests that could potentially create accounts (if mocks failed to work),
  please use `test@amazon.com` instead.

When you want to write documentation/tests and need an example account id, the
following account ids may be used:

* `111111111111`
* `222222222222`
* ... to ...
* `999999999999`
* or to show the length more easily:
* `012345678910`
* `012345671234`
* `123456789012`

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/tests/stubs/stub_deployment_map.yml [hunk 1]
```diff
@@ -1,230 +0,0 @@
-pipelines:
```
Comment (sbkok) [inline src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/tests/stubs/stub_deployment_map.yml:1]: The content of the file was an out of date copy of the [`${bootstrap-repo}/adf-bootstrap/deployment/lambda_codebase/initial_commit/pipelines_repository/example-deployment_map.yml`](https://github.com/awslabs/aws-deployment-framework/blob/991a076ec70972acebeaf45c6a40de9f8ec681a4/src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/initial_commit/pipelines_repository/example-deployment_map.yml)

So instead, I made this a symbolic link so they cannot go out of sync from now on.
See the next file in this review, with a similar name. Unfortunately GitHub doesn't display this diff clearly.

**Review by StewartW — APPROVED** [review]
LGTM. Thanks! :shipit:

File: src/lambda_codebase/initial_commit/bootstrap_repository/example-adfconfig.yml [hunk 2]
```diff
@@ -8,13 +8,13 @@ regions:
...
       action: safe  # Can be safe or remove-base (see docs)
   scp:
-    keep-default-scp: enabled  # determines if the default AWSFullAccess SCP stays attached to all OU's
-  scm: # Source control management
-    auto-create-repositories: enabled # if true and using codecommit as source the repository will be automatically created
-    default-scm-branch: master        # allow to specify custom default branch if a pipeline specific configuration is not provided. If the parameter is not specified it defaults to the "master" branch.
+    keep-default-scp: enabled  # Determines if the default AWSFullAccess SCP stays attached to all OU's
+  scm:  # Source control management
+  auto-create-repositories: enabled  # If true and using CodeCommit as source, the repository will be automatically created
+    default-scm-branch: master       # The default branch is used when the pipeline does not specify a specific branch. If this parameter is not specified, it defaults to the "master" branch.
```
Comment (thomasmcgannon) [inline src/lambda_codebase/initial_commit/bootstrap_repository/example-adfconfig.yml:20]: Should this be main instead of master now?

Comment (sbkok) [reply src/lambda_codebase/initial_commit/bootstrap_repository/example-adfconfig.yml:20]: It will be, but not as part of this PR :smile:

**Review by thomasmcgannon — DISMISSED** [review]
Lots of dedication to cleaning up :D lgtm!

**Review by StewartW — APPROVED** [review]

**Review by thomasmcgannon — APPROVED** [review]

**Pull request merged**